src/decorators/debounce.ts

```diff
@@ -1,6 +1,12 @@
-import lodashDebounce, {DebounceSettings} from 'lodash/debounce';
+import debounce from 'lodash/debounce';
 
-export default function debounce(wait = 500, options?: DebounceSettings) {
+export interface DebounceSettings {
```

Comment:
@lemonmade Do you think we can do `import {DebounceSettings} from 'lodash';` @michelleyschen did a `yarn build` and it does not include the entire `lodash` library that I was worried about